### DISCUSSION TITLE
Custom iptables path

### DISCUSSION THREAD

**Discussion by hagen-bauer-regioit** [thinkst/opencanary#277] [Feature Requests]
On SLES iptables is in 
```
which iptables
/usr/sbin/iptables
```
which leads to the situation that  the firewall rules for logging are not created. 

It would be better if the path is checked.

**Comment by jayjb** [comment]
Hi @hagen-bauer-regioit,

Thanks for reporting this. We are aiming to be as compatible across different OSs as possible. We would really appreciate a PR doing what you suggest 😄

**Comment by hagen-bauer-regioit** [comment]
I would already have done it if my python skills would be good enough ;-)

**Comment by jayjb** [comment]
Closing this discussion as it has been implemented in #297
